parser/utility.py

```diff
@@ -8,3 +9,20 @@ def save_list_of_apartments(apartments: list, file_path: str) -> None:
         json_file.write(json.dumps(apartments, indent=4))
 
 
+def suppress(exception: Exception, error_msg: str):
```

Comment:
Just one suggestion for possible improvement (no need to do this right now). decorator can actually take a tuple of exceptions rather then single exception. `def suppress(exceptions: Tuple[Exception], msg: Optional[str] = None): ...`.
`except exceptions as e:`. So basically except can take a tuple of exceptions as well, but `e` will be assigned to a single one that was raised.